### PR TITLE
Unwildcard python files and major code cleanup

### PR DESCRIPTION
**Pull request by mstabrin**
@shadowwalkersb Could you help me with the continuous integration fix?
I tried to fix the python -m compileall errors, but that did unfortunately not work.

### CODE REVIEW

**Comment by shadowwalkersb** [comment]
> I tried to fix the python -m compileall errors, but that did unfortunately not work.

How about just excluding `unused`?
`python -m compileall -q -x unused .`

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# EMAN2 libs
```
Comment (shadowwalkersb) [inline .gitignore]: Was this uncommented intentionally?

Comment (mstabrin) [reply .gitignore]: No this was an accident, it is back to normal

**Comment by mstabrin** [comment]
Yes, we can do that.


The problem was, that I had a branch named something.py, which was treated by the compileall command like a normal python file.

It occurred on multiple systems, but now it turns out that fresh installations have no conflict with this branch I created some time ago and this one is deleted already.

Therefore ignoring the unused folder should be enough.


Gonna try this right now!


best,

Markus


________________________________
Von: shadow_walker <notifications@github.com>
Gesendet: Freitag, 7. Dezember 2018 05:45
An: cryoem/eman2
Cc: Markus Stabrin; Author
Betreff: Re: [cryoem/eman2] Unwildcard python files and major code cleanup (#280)


I tried to fix the python -m compileall errors, but that did unfortunately not work.

How about just excluding unused?
python -m compileall -q -x unused .

-

**Comment by mstabrin** [comment]
obsolete

**Comment by shadowwalkersb** [comment]
Can you, please, delete the branch if not needed anymore?

**Pull request closed without merging**